debug/org.eclipse.debug.ui/css/e4-dark_debug_prefstyle.css

```diff
@@ -57,4 +57,5 @@ IEclipsePreferences#org-eclipse-debug-ui {
 
 #VariablesViewer {
     font-family: '#org-eclipse-debug-ui-VariableTextFont';
+    background-color: #2F2F2F;
```

Comment:
this change causes a side effect: The tree in the variables view does then no longer have the zebra 🦓 lines.

Reply:
The interesting thing is that e.g. JDTs problems view also uses this zebra feature but does not have this flickering issue.